hera_cal/reflections.py

```diff
@@ -758,8 +768,8 @@ def subtract_model(self, data, keys=None, overwrite=False, ifft=True, ifftshift=
             self.pcomp_model_fft[k] = model_fft
             self.data_pcmodel_resid[k] = data[k] - model_fft
 
-    def interp_u(self, umodes, times, full_times=None, uflags=None, keys=None, overwrite=False,
-                 gp_frate=1.0, gp_frate_degrade=0.0, gp_nl=1e-12, kernels=None, optimizer=None, verbose=True):
+    def interp_u(self, umodes, times, full_times=None, uflags=None, keys=None, overwrite=False, Ninterp=None,
```

Comment:
To be pedantic, I'd say this is really a "fit" rather than an interpolation. Interpolation goes exactly through the known points, but this won't in general.

Reply:
okay. I'm fine w/ the nomenclature though